index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"><img src="" alt=""> Canopy</li>
+            <li> <img src="images/Counter.png" alt=""> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt=""> 2015</li>
```

Comment:
- [ ] Kindly add `alt` attribute to all your `img` elements. The `alt` attribute provides alternative information for an image if a user for some reason cannot view it; ℹ️